datameta/api/submissions.py

```diff
@@ -94,7 +98,7 @@ def validate_submission_association(db_files, db_msets):
     """
     errors = []
     # Collect files with no data associated
-    errors += [ (db_file, None, "No data uploaded") for file_id, db_file in db_files.items() if db_file.content_uploaded==False ]
+    errors += [ (db_file, None, "No data uploaded") for file_id, db_file in db_files.items() if db_file.content_uploaded is False ]
```

Comment:
can this be a `not db_file.content_uploaded` or can `db_file.content_uploaded` be `None`?

Reply:
That would work, too.